code/datums/status_effects/debuffs/slime/slime_food.dm

```diff
@@ -0,0 +1,65 @@
+///Adds pheromones to a mob. If the target slime drains the mob to death, they might befriend the user
+/datum/status_effect/slime_food
+	id = "slime_food"
+	var/befriend_chance = 30
+	var/mob/living/carbon/human/feeder
+	alert_type = null
+
+/datum/status_effect/slime_food/on_creation(mob/living/new_owner, mob/living/carbon/human/feeder, befriend_chance = 100)
+	src.befriend_chance = befriend_chance
+	src.feeder = feeder
+	return ..()
+
+/datum/status_effect/slime_food/on_apply()
+	if(isnull(feeder))
+		return FALSE
+
+	if(!ishuman(feeder)) //don't give the AI pheromones
+		return FALSE
+
+	RegisterSignals(feeder, list(COMSIG_QDELETING), PROC_REF(on_feeder_deleted))
+	RegisterSignals(owner, list(COMSIG_COMPONENT_CLEAN_ACT), PROC_REF(on_feeder_deleted))
+	RegisterSignals(owner, list(COMSIG_SLIME_DRAINED), PROC_REF(on_drained))
+	RegisterSignals(owner, list(COMSIG_ATOM_EXAMINE), PROC_REF(on_examine))
```

Comment:
```suggestion
	RegisterSignal(feeder, COMSIG_QDELETING, PROC_REF(on_feeder_deleted))
	RegisterSignal(owner, COMSIG_COMPONENT_CLEAN_ACT, PROC_REF(on_feeder_deleted))
	RegisterSignal(owner, COMSIG_SLIME_DRAINED, PROC_REF(on_drained))
	RegisterSignal(owner, COMSIG_ATOM_EXAMINE, PROC_REF(on_examine))
```

`RegisterSignals()` exists for a very specific reason, which this is not

Reply:
I think I was extremly sleepy when I did this, and copied the wrong line.